features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/data/APIValidation.js

```diff
@@ -53,13 +53,43 @@ const userRoleSchema = Joi.extend(joi => ({
     ],
 }));
 
+const apiSchema = Joi.extend(joi => ({
+    base: joi.string(),
+    name: 'api',
+    rules: [
+        {
+            name: 'isAPIParameterExist',
+            validate(params, value, state, options) { // eslint-disable-line no-unused-vars
+                const api = new API();
+                return api.validateAPI(value);
+            },
+        },
+    ],
+}));
+
+const documentSchema = Joi.extend(joi => ({
+    base: joi.string(),
+    name: 'document',
+    rules: [
+        {
+            name: 'isDocumentPresent',
+            validate(params, value, state, options) { // eslint-disable-line no-unused-vars
```

Comment:
Shall fix this eslint issue rather than disabling the rule. I think there is an unused variable here